squidlib-util/src/test/java/squidpony/squidmath/LightRNGTest.java

```diff
@@ -14,6 +14,7 @@
  *
  */
 public class LightRNGTest {
+	private static final double DELTA = 0.00000000001;
```

Comment:
@tommyettinger you could use it this way (as a reference to your lates comment in #79